lib/logstash/outputs/elasticsearch.rb

```diff
@@ -33,6 +33,24 @@
 # If using the default `protocol` setting ("node"), your firewalls might need
 # to permit port 9300 in *both* directions (from Logstash to Elasticsearch, and
 # Elasticsearch to Logstash)
+#
+# ## Retry Policy
+#
+# Not all events always make it successfully to Elasticsearch, and they are not always 
```

Comment:
Change it to: "By default all bulk requests to ES are synchronous. Not all events always make it successfully ..